x/programs/wasmlanche/src/host.rs

```diff
@@ -0,0 +1,258 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+use crate::HostPtr;
+use cfg_if::cfg_if;
+
+#[cfg(feature = "test")]
```

Comment:
Can you throw all the `cfg(feature = "test")` stuff into an inline `mod`, then do a:
```rust
#[cfg(feature = "test")]
mod test_stuff {
    // please pick a better name
    // ...
}

#[cfg(feature = "test")]
pub use test_stuff::*;
```